lib/src/components/NxTextLink/__tests__/NxTextLink.test.tsx

```diff
@@ -5,114 +5,106 @@
+    expect(renderEl()).toHaveAttribute('rel', '');
+    expect(renderEl({ rel: 'foo' })).toHaveAttribute('rel', 'foo');
 
-    expect(getShallow({ noReferrer: true })).toHaveProp('rel', 'noreferrer');
+    expect(renderEl({ noReferrer: true })).toHaveAttribute('rel', 'noreferrer');
 
-    expect(getShallow({ noReferrer: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
-    expect(getShallow({ noReferrer: true, rel: 'foo' }).prop('rel')).toContain('foo');
+    expect(renderEl({ noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
+    expect(renderEl({ noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
   });
 
   it('adds the noreferrer rel if the external prop is true and noReferrer is not false', function() {
-    expect(getShallow({ external: true })).toHaveProp('rel', 'noreferrer');
+    expect(renderEl({ external: true })).toHaveAttribute('rel', 'noreferrer');
 
-    expect(getShallow({ external: true, rel: 'foo' }).prop('rel')).toContain('foo');
-    expect(getShallow({ external: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
+    expect(renderEl({ external: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
+    expect(renderEl({ external: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
 
-    expect(getShallow({ external: true, noReferrer: true })).toHaveProp('rel', 'noreferrer');
-    expect(getShallow({ external: true, noReferrer: false })).toHaveProp('rel', '');
+    expect(renderEl({ external: true, noReferrer: true })).toHaveAttribute('rel', 'noreferrer');
+    expect(renderEl({ external: true, noReferrer: false })).toHaveAttribute('rel', '');
 
-    expect(getShallow({ external: true, noReferrer: true, rel: 'foo' }).prop('rel')).toContain('noreferrer');
-    expect(getShallow({ external: true, noReferrer: true, rel: 'foo' }).prop('rel')).toContain('foo');
+    expect(renderEl({ external: true, noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('noreferrer');
+    expect(renderEl({ external: true, noReferrer: true, rel: 'foo' })?.getAttribute('rel')).toContain('foo');
   });
 
   it('sets aria-disabled on the text link when the disabled prop is set', function() {
-    expect(getShallow()).toHaveProp('aria-disabled', false);
-    expect(getShallow({ disabled: true })).toHaveProp('aria-disabled', true);
+    expect(renderEl()).toHaveAttribute('aria-disabled', 'false');
+    expect(renderEl({ disabled: true })).toHaveAttribute('aria-disabled', 'true');
   });
 
   it('sets link role on the text link when the disabled prop is set', function() {
-    expect(getShallow()).not.toHaveProp('role', 'link');
-    expect(getShallow({ disabled: true })).toHaveProp('role', 'link');
+    expect(renderEl()).not.toHaveAttribute('role', 'link');
+    expect(renderEl({ disabled: true })).toHaveAttribute('role', 'link');
   });
 
   it('does not pass href value on the text link when the disabled prop is set', function() {
```

Comment:
Implementation detail.  What we really want to check is whether clicking the link does anything when it is/isn't disabled

Reply:
> Implementation detail. What we really want to check is whether clicking the link does anything when it is/isn't disabled

Something like this? b72d4c0c3aaf75b02b9195688d8c0b38d231ad7f

Reply:
> Something like this? https://github.com/sonatype/sonatype-react-shared-components/commit/b72d4c0c3aaf75b02b9195688d8c0b38d231ad7f

Nevermind, after a little bit of research it doesn't look like JSDOM really implements enough to verify that links can truly be followed.  Checking the `href` attribute seems like the best approach afterall.  What you had originally was good.

Reply:
> > Something like this? [b72d4c0](https://github.com/sonatype/sonatype-react-shared-components/commit/b72d4c0c3aaf75b02b9195688d8c0b38d231ad7f)
> 
> Nevermind, after a little bit of research it doesn't look like JSDOM really implements enough to verify that links can truly be followed. Checking the `href` attribute seems like the best approach afterall. What you had originally was good.

Changed it: 6748453f2635cc4c13e839c3db1dac9ea6ed782c